Performance Plots 2.0 update

athompson-git opened the pull request:
1) Adding instructions in the README for Performance Plots 2.0
2) Minor typo fixes in the README.
3) Remove oldTTrees support.
4) Fix Gaussian fit colors in functions.py

athompson-git committed:
Update README.md

athompson-git committed:
Update README.md

athompson-git committed:
Update README.md

athompson-git committed:
Update README.md

athompson-git committed:
Update README.md

athompson-git committed:
Applying Band-aid for oldTTrees

athompson-git committed:
Adding oldTTrees band-aid

athompson-git committed:
Update Step1_make_2d_plots.py

athompson-git committed:
Update Step1_make_2d_plots_histogrammar.py

lpernie commented:
Generally CAF can run ~900 jobs at the same time. For this reason I generally split in ~800 jobs. No more than 900 cause the last jobs will need to wait for other jobs to finish, not much less then 900 because more jobs are parallelized more are quick.
500 is fine too, as you prefer.

athompson-git committed:
Update README.md

athompson-git committed:
Update README.md

athompson-git committed:
Update README.md

athompson-git commented:
Done, I've updated it to give the recommended range.

athompson-git committed:
Update Step1_make_2d_plots.py

athompson-git committed:
Update Step1_make_2d_plots_histogrammar.py

athompson-git committed:
removing oldTTrees

athompson-git committed:
Update Step1_make_2d_plots_histogrammar.py

athompson-git committed:
Remove oldTTrees functionality

athompson-git committed:
removed oldTTrees functionality

athompson-git committed:
Fix coloring for gaussian fits

Fix coloring for gaussian fits. For now we shouldn't have to worry about line color for the case fitType != "gaus" since that case just corresponds to making TH2F's of chi^2 with error bars.

athompson-git committed:
Update Step2_makeProfiles.py
